### PR TITLE
Add copy for the Wellbeing special edition, and a date string helper type

### PR DESCRIPTION
**Pull request by i-hardy**
## What are you doing in this PR?

This adds copy for an upcoming special edition, as well as a helper type for type-safe YYYY-MM-DD date strings.

[**Trello Card**](https://trello.com/c/NBE0xhsE)

## Why are you doing this?

There's a new special edition coming out tomorrow which we want to be able to promote as normal. The helper type adds a little extra clarity and type-safety to setting up timed copy in future as date strings always need to be in a specific format, and the type should be reusable elsewhere in the codebase.

## Screenshots
Screenshot 2022-04-13 at 11 35 57

### CODE REVIEW

**Comment by i-hardy** [comment]
[Promote new Special Edition on the Digital Subscription product page (14 April)](https://trello.com/c/NBE0xhsE/390-promote-new-special-edition-on-the-digital-subscription-product-page-14-april)

File: support-frontend/assets/helpers/types/DateString.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+type Digit = 0 | OneToNine;
+
+type YYYY = `20${Digit}${Digit}`;
+
+type MM = `0${OneToNine}` | `1${0 | 1 | 2}`;
+
+type DD = `${0}${OneToNine}` | `${1 | 2}${Digit}` | `3${0 | 1}`;
+
+export type DateYMDString = `${YYYY}-${MM}-${DD}`;
```
Comment (TomPretty) [inline support-frontend/assets/helpers/types/DateString.ts:11]: so cool!

**Review by TomPretty — APPROVED** [review]
LGTM 👍 love the new type!

File: support-frontend/assets/helpers/types/DateString.ts (hunk 1 above)
Comment (rupertbates) [inline support-frontend/assets/helpers/types/DateString.ts:11]: Nice! I guess technically it's a 21st century date string but that's fine.

**Review by rupertbates — APPROVED** [review]
👍

**Pull request merged**